xapian-core/include/xapian/cluster.h

```diff
@@ -449,7 +449,7 @@ class XAPIAN_VISIBILITY_DEFAULT ClusterSet {
     /// Clear all the clusters in the ClusterSet
     void clear_clusters();
 
-    /** Recalculate the centroids for all the centroids
+    /** Recalculate the centroid for all the clusters
```

Comment:
Should really be either "Recalculate the centroids for all the clusters [...]" or "Recalculate the centroid for each of the clusters [...]" (perhaps better as it captures that there's one centroid per cluster) - as currently written it's implying that there's just one centroid.